### PR TITLE
[IMP] recruitment: improve usability

### PR DESCRIPTION
**Pull request by tivisse**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_recruitment/models/hr_job.py [hunk 1]
```diff
@@ -82,8 +82,8 @@ def _get_first_stage(self):
         self.ensure_one()
         return self.env['hr.recruitment.stage'].search([
             '|',
-            ('job_id', '=', False),
-            ('job_id', '=', self.id)], order='sequence asc', limit=1)
+            ('job_ids', '=', False),
+            ('job_ids', 'in', self.id)], order='sequence asc', limit=1)
```
Comment (kos94ok-3D) [inline addons/hr_recruitment/models/hr_job.py]: `('job_ids', 'in', self.id)`
Why do you use 'in' instead of '=' for id?

**Comment by tivisse** [comment]
@kba-odoo FYI The comments are already applied on your branch. 

Thanks.

**Comment by tivisse** [comment]
@robodoo r+

File: addons/hr_recruitment/models/hr_recruitment.py [hunk 2]
```diff
@@ -457,11 +483,11 @@ def create_employee_from_applicant(self):
                     'work_phone': applicant.department_id and applicant.department_id.company_id
                             and applicant.department_id.company_id.phone or False})
                 applicant.write({'emp_id': employee.id})
-                applicant.job_id.message_post(
-                    body=_('New Employee %s Hired') % applicant.partner_name if applicant.partner_name else applicant.name,
-                    subtype="hr_recruitment.mt_job_applicant_hired")
```
Comment (tivisse) [inline addons/hr_recruitment/models/hr_recruitment.py]: ``subtype`` is not a valid argument, leading to a warning on runbot.

The correct way to handle this is 
```python
subtype_id=self.env.ref("hr_recruitment.mt_applicant_hired").id
```

``message_post_with_view`` has not the same signature than ``message_post`` by the way.

Comment (tivisse) [reply addons/hr_recruitment/models/hr_recruitment.py]: Wrong PR, my bad.

**Comment by tivisse** [comment]
@robodoo r+

**Pull request closed without merging**